src/main/java/net/minecraftforge/common/util/LazyOptional.java

```diff
@@ -295,4 +295,4 @@ public void invalidate()
         this.isValid = false;
         this.listeners.forEach(e -> e.accept(this));
     }
-}
+}
```

Comment:
please undo this minor change

Reply:
Eclipse seems to want to keep removing the newline character, I had to add it manually with Notepad++.